app/Http/Controllers/Admin/NewsController.php

```diff
@@ -29,4 +30,45 @@ public function index()
                     ->orderby('id', 'ASC')->paginate(News::ROW_LIMIT);
         return view('backend.news.index', compact('news'));
     }
+
+    /**
+     * Display form edit a News.
+     *
+     * @param string $slug of News
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($slug)
+    {
+        $columns = [
+                    'id',
+                    'title',
+                    'content',
+                    'category_id'
+        ];
+        $news = News::where('slug', $slug)
+                    ->select($columns)
+                    ->get();
```

Comment:
Edit only 1 news so why you use `get` function, that function return an collection. Use `firstOrFail` instead